src/test/java/org/gridsuite/study/server/SensitivityAnalysisTest.java

```diff
@@ -731,4 +736,24 @@ public void testSensitivityAnalysisParameters() throws Exception {
                 status().isOk(),
                 content().string(SENSITIVITY_ANALYSIS_UPDATED_PARAMETERS_JSON));
     }
+
+    @Test
+    public void testGetComputationCount() throws Exception {
```

Comment:
testGetSensitivityAnalysisParametersComplexity

Reply:
It's a proposal from Slimane